src/core/util.c

```diff
@@ -348,7 +348,13 @@ util_init(void)
 #endif
 
 #if VG_DRD_ENABLED || VG_HELGRIND_ENABLED
```

Comment:
You dont need outer if and initialisation with 0 (as it is a global varable )

Reply:
Done